x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,35 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Decimal,  // price in fiat currency show to user
+  fiat_currency String,  // fiat currency name
+  token_denom String,  // denomation of token
+  conversion_rate Decimal,  // conversion rate from fiat currency to token, maintained by owner
```

Comment:
probably good for @christopher-crypto to comment on this one -- maybe it'll be more practical to submit it each time with the collection message (so that there's less delay between rate setting and collection)?

Reply:
used fixed denomination now.